tests/integration_test.py

```diff
@@ -10,140 +38,231 @@
 	TEST_DID = "did:plc:bwxddkvw5c6pkkntbtp2j4lx"
 	TEST_HANDLE = "local.dev.retr0.id"
 	TEST_PASSWORD = "lol"
+TEST_PRIVKEY = crypto.keygen_p256()
+
+@pytest.fixture
+async def test_pds(aiolib):
+	queue = asyncio.Queue()
+	with tempfile.TemporaryDirectory() as tempdir:
+		db_path = f"{tempdir}/millipds-0000.db"
+		db = database.Database(path=db_path)
```

Comment:
I believe passing `:memory:` will make sqlite use an in-memory db, which would avoid the need for a temp path. But, I suppose tests could plausibly involve writing larger-than-memory amounts of data in the future, so let's keep it.